weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/compute_graph_interface.ts

```diff
@@ -0,0 +1,793 @@
+/**
```

Comment:
This file is largely just a copy of functions that previously existed in `interface.py`. Almost no logic actually changed here.